src/index.tsx

```diff
@@ -66,6 +66,8 @@ export { default as SelectInput } from './components/SelectInput';
 export type { Props as SelectInputProps } from './components/SelectInput';
 export type { Props as TabProps } from './components/Tab';
 export { default as Tab } from './components/Tab';
+export type { Props as TableProps } from './components/Table';
+export { default as Table } from './components/Table';
```

Comment:
We need to export more than the Table component

Reply:
Done!